toolkit/resources/manifests/package/pkggen_core_x86_64.txt

```diff
@@ -74,9 +74,9 @@ xz-5.4.4-1.azl3.x86_64.rpm
 xz-devel-5.4.4-1.azl3.x86_64.rpm
 xz-lang-5.4.4-1.azl3.x86_64.rpm
 xz-libs-5.4.4-1.azl3.x86_64.rpm
-zstd-1.5.5-2.azl3.x86_64.rpm
-zstd-devel-1.5.5-2.azl3.x86_64.rpm
-zstd-libs-1.5.5-2.azl3.x86_64.rpm
+zstd-1.5.5-5.azl3.x86_64.rpm
+libzstd-1.5.5-5.azl3.x86_64.rpm
```

Comment:
This might not be a big deal, but just noting: When building the worker chroot, the packages are installed in the order listed in this file. With this change the zstd libs and devel package install order is swapped. Probably not a big deal but it would be safest to preserve the order (`zstd` -> `libzstd-devel` -> `libzstd`)

Reply:
oh wow this lists packages to install entirely separately/individually and in order? I thought it was mostly just alphabetical. Why is each rpm installed separately instead of just passing them all to rpm (or dnf) and letting rpm work out the dependency details?

Reply:
ok sorted them in order or dependence. i'm still surprised that the user of this file requires that.

Reply:
> oh wow this lists packages to install entirely separately/individually and in order? I thought it was mostly just alphabetical. Why is each rpm installed separately instead of just passing them all to rpm (or dnf) and letting rpm work out the dependency details?

I share your concern here (and I'm also personally annoyed by this in addition to being concerned). This is one of the things we want to fix as part of either completely dropping the manifest files or at least making them "versionless"/easier to use. We have that work item scheduled for later.